zzzz_modular_occulus/code/modules/mob/living/simple_animal/hostile/siren/composer.dm

```diff
@@ -0,0 +1,92 @@
+/mob/living/simple_animal/hostile/siren/composer
+	name = "composer"
+	desc = "A six legged creature, it is a mesh of organic and synthetic material, chittering strange noises, and regurgitating things it's heard."
+	icon = 'zzzz_modular_occulus/icons/mob/siren/composer.dmi'
+	icon_state = "sixleg"
+	icon_living = "sixleg"
+	icon_aggro = "sixleg"
+	icon_dead = "sixleg-dead"
+	icon_gib = "syndicate_gib"
+	mouse_opacity = 2
+	move_to_delay = 5
+	ranged = 0
+	vision_range = 5
+	density = TRUE
+	aggro_vision_range = 11
+	idle_vision_range = 9
+	speed = 4
+	maxHealth = 80
+	health = 80
+	harm_intent_damage = 25
+	melee_damage_lower = 20
```

Comment:
These guys hit really hard if you let them. Might want to tone it down just a tad. 15-20?

Reply:
Can do that

Reply:
You didn't actually do the requested change.